### PR TITLE
website with navigation project

### PR DESCRIPTION
**Pull request by sambeck87**
Navbar was created
All sections were divided (there are three different sections to show) All sections were stylized
The webpage was created dynamically with JavaScript

### CODE REVIEW

File: script_class.js [hunk 1]
```diff
@@ -97,3 +97,51 @@ deleteBtn.forEach((btn) => {
...
+function List() {
+  list.classList.remove('hide');
+  subtitle.classList.add('hide');
+  form.classList.add('hide');
+  contact.classList.add('hide');
+  btnList.classList.add('blue');
+  btnList.classList.remove('link_nav');
+  btnAddNew.classList.add('link_nav');
+  btnContact.classList.add('link_nav');
+}
```
Comment (ha-manel) [inline script_class.js:121]: - [ ] When the app starts both the list and the form are shown at the same time while the navigation says I'm on the list section, kindly fix this issue so only the list shows when the app is started 👍 

image

File: index.html [hunk 2]
```diff
@@ -10,21 +10,47 @@
...
+      <h1>All awesome books</h1>
+      <div id="books"></div>
+    </div>
   </div>
-  <hr class="divisor">
 
-  <h2 class="new_book">Add a new book</h2>
-  <form action="#" class="form">
+  <h2 id="subtitle" class="new_book hide">Add a new book</h2>
+  <form id="add-books" action="#" class="form >
```
Comment (ha-manel) [inline index.html]: - [ ] in the wireframe and in the project requirements, each book should have an author and a title. Kindly add the title input field to your form 👍 I think the issue is in the missing `"` of the class attribute of the form element. It also caused the problem with adding and viewing the books list 😅 

image

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi Team👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Professional README file ✔️

## Required Changes:
- [ ] Kindly follow best practices when you write your commit messages, they should be capitalized sentences in the imperative form like "Add linters" instead of "linters added" 💯 
- [ ] The website should be fully functional but I can't add books or view the list. Kindly fix this issue 👍

Animation
 
_Kindly check the comments below_


------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**